src/db/tables/subscribers.js

```diff
@@ -196,6 +196,34 @@ async function setOnerepProfileId (subscriber, onerepProfileId) {
     })
 }
 
+/**
+ * @param {number} onerepProfileId
+ * @param {number} oneRepScanId
+ */
+async function setOnerepScan (onerepProfileId, oneRepScanId) {
+  await knex('onerep_scans')
+    .insert({
+      onerep_profile_id: onerepProfileId,
+      onerep_scan_id: oneRepScanId,
+      created_at: knex.fn.now()
+    })
+}
+
+/**
+ * @param {number} onerepProfileId
+ * @param {number} oneRepScanId
+ * @param {object} oneRepScanResults
+ */
+async function setOnerepScanResults (onerepProfileId, oneRepScanId, oneRepScanResults) {
+  await knex('onerep_scans')
+    .where('onerep_profile_id', onerepProfileId)
+    .andWhere('onerep_scan_id', oneRepScanId)
+    .update({
+      onerep_scan_results: oneRepScanResults,
+      updated_at: knex.fn.now()
+    })
+}
```

Comment:
```suggestion
/**
 * @param {number} oneRepProfileId
 * @param {number} oneRepScanId
 * @param {object} oneRepScanResults
 */
async function setOneRepScanResults (oneRepProfileId, oneRepScanId, oneRepScanResults) {
  await knex('onerep_scans')
    .where('onerep_profile_id', oneRepProfileId)
    .andWhere('onerep_scan_id', oneRepScanId)
    .update({
      onerep_scan_results: oneRepScanResults,
      updated_at: knex.fn.now()
    })
}
```

PROTIP: I also think you can pass an object into the Knex.js `where` clause instead of chained `.where().andWhere()`; https://knexjs.org/guide/query-builder.html#where:

```js
knex('users')
  .where({
    first_name: 'Test',
    last_name:  'User'
  })
  .select('id')
```

Reply:
Yeah, I know I'm doing something wrong when the caller needs to do stuff like `result[0].blah` :)